finnish/README.md

```diff
@@ -1,9 +1,26 @@
-# Finish Braille Specifications and Guidelines
+# Finnish Braille Specifications and Guidelines
 
-## [Pistekirjoituksen perusteet](pistek_perusteet.html)
+## [Pistekirjoituksen perusteet (PDF)](Pistekirjoituksen_perusteet.pdf)
 
-- _by_: Finnish Braille Authority and the Finnish Library for the Visually Impaired
+- _description_: 6-dot braille for literary texts
+- _by_: Finnish delegation for braille and Accessibility Library Celia
 - _applies to_: Finland
-- _published_: 2000
-- _language_: Finnish
+- _published_: 2021
+- _language_: Finnish, Finland Swedish
 
+## [Matematiikan, fysiikan ja kemian pistemerkinnät (PDF)](Matematiikan_fysiikan_ja_kemian_pistemerkinnät.pdf)
+
+- _description_: 6-dot braille for math, physics and chemistry texts
+- _by_: Finnish delegation for braille and Accessibility Library Celia
+- _applies to_: Finland
+- _published_: 2022
+- _language_: Finnish, Finland Swedish
```

Comment:
Is this the language of the document, or the languages to which the braille code applies?

Reply:
In case of the latter (which I'm guessing it is), the documentation should become:

    - _language_: Finnish
    - _applies to_: Finland (Finnish, Finland Swedish)

and in the library itself, it would be good if the following metadata would be added to the Finnish tables: 

    #+language: sv-FI